simplified-tests/src/test/java/org/nypl/simplified/tests/books/controller/BooksControllerContract.kt

```diff
@@ -898,6 +969,139 @@ abstract class BooksControllerContract {
     Assertions.assertEquals(statusBefore, statusAfter)
   }
 
+  @Test
+  @Timeout(value = 3L, unit = TimeUnit.SECONDS)
+  fun testBooksPreviewSuccess() {
+    val controller =
+      createPreviewController(
+        exec = this.executorBooks,
+        feedExecutor = this.executorFeeds,
+        accountEvents = this.accountEvents,
+        profileEvents = this.profileEvents,
+        http = this.lsHTTP,
+        profiles = this.profiles,
+        accountProviders = MockAccountProviders.fakeAccountProviders(),
+        patronUserProfileParsers = this.patronUserProfileParsers
+      )
+
+    this.server.enqueue(
+      MockResponse()
+        .setResponseCode(200)
+        .setBody(this.simpleUserProfile())
+    )
+
+    this.server.enqueue(
+      MockResponse()
+        .setResponseCode(200)
+        .setBody(Buffer().readFrom(resource("testBooksDelete.xml")))
```

Comment:
Is this the correct response? The filename is confusing.

Reply:
I tried using any file, but I don't need it indeed